### PR TITLE
RWA-1695 - added unit tests

### PR DESCRIPTION
**Pull request by mcareyma**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RWA-1695

### Change description ###

Added tests for full coverage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by mrmsali — APPROVED** [review]
Looks good to me.

**Review by chrisdavecm — APPROVED** [review]

File: src/test/unit/task-moniter-service.test.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import { TaskMonitorService } from '../../main/services/task-monitor-service';
```
Comment (raghera) [inline src/test/unit/task-moniter-service.test.ts:1]: The filename has a typo - `moniter` should be `monitor`

File: src/test/unit/logger.test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const exceptionSpy = jest.spyOn(logger, 'exception').mockImplementation(() => {console.log('foo');});
+    logger.exception('foo','info');
+    expect(exceptionSpy).toHaveBeenCalled(); 
+  });
+  it('logs console',()=>{
+    logger.console('foo','info');
+    const consoleSpy = jest.spyOn(logger, 'console').mockImplementation(() => {console.log('foo');});
+    logger.console('foo','info');
+    expect(consoleSpy).toHaveBeenCalled(); 
+  });
```
Comment (raghera) [inline src/test/unit/logger.test.ts:29]: Not sure why have these tests .. but since you didn't write them and they seem to pass we can keep them for now.

**Review by raghera — COMMENTED** [review]
Tests seem to be running in preview and doesn't seem to matter that the s2s url is always localhost since whenever a response is required there's a handler that provides what the unit test needs.

Can probably refine these tests later.

File: src/test/unit/task-moniter-service.test.ts (hunk 1 above)
Comment (SammiChong) [reply src/test/unit/task-moniter-service.test.ts]: Updated

**Pull request merged**